packages/nimble-components/src/table-column/base/models/column-internals.ts

```diff
@@ -163,14 +174,27 @@ export class ColumnInternals<TColumnConfig> {
     @observable
     public currentSortDirection: TableColumnSortDirection = TableColumnSortDirection.none;
 
-    public constructor(options: ColumnInternalsOptions) {
+    public readonly validator: TColumnValidator;
+
+    public constructor(options: ColumnInternalsOptions<TColumnValidator>) {
         this.cellRecordFieldNames = options.cellRecordFieldNames;
         this.cellViewTemplate = createCellViewTemplate(options.cellViewTag);
         this.groupHeaderViewTemplate = createGroupHeaderViewTemplate(
             options.groupHeaderViewTag
         );
         this.delegatedEvents = options.delegatedEvents;
         this.sortOperation = options.sortOperation ?? TableColumnSortOperation.basic;
+        this.validator = options.validator;
+
+        const notifier = Observable.getNotifier(this.validator);
+        notifier.subscribe(this);
+    }
+
+    /** @internal */
+    public handleChange(source: unknown, args: unknown): void {
+        if (source instanceof ColumnValidator && args === 'isColumnValid') {
+            this.validConfiguration = this.validator.isColumnValid;
+        }
     }
```

Comment:
Thinking about it again, this indirection seems unnecessary.  The columnIInternals.validConfiguration property was the observable property representing the validator state. Now the validator has its own observable property, so I don't see why columnInternals needs the observable validConfiguration property anymore. Things should just check columnInternals.validator.isColumnValid's observable prop.